src/js/constants/Viewports.js

```diff
@@ -0,0 +1,7 @@
+// We use this value in the css
+// when targeting tablet or mobile
+export const MOBILE_THRESHOLD = 994;
+
+export const DESKTOP = "DESKTOP";
```

Comment:
Was there a reason not to chose enum here? Wonder which one fits better 🤔

Reply:
When @Fabs and I paired we like how declarative this option is.